examples/simple-game-server/main.go

```diff
@@ -523,11 +524,15 @@ func getPlayerCapacity(s *sdk.SDK) string {
 }
 
 // playerConnect connects a given player
-func playerConnect(s *sdk.SDK, id string) {
+func playerConnect(s *sdk.SDK, id string) string {
```

Comment:
I changed this because of the issue https://github.com/googleforgames/agones/issues/1957
> What you expected to happen:
> Should not panic, and return both ok with a false value and the error.

The panic is fixed from the previous PR.
Then, I made this change to send some information which the player is connected or not(true or false) to the client.

Reply:
Currently, the server runs log.Fatalf when it can't connect a player.
As you said, I think it is okay to use log.Printf without return true/false.